scripts/make-in-container.sh

```diff
@@ -46,4 +46,4 @@ build_dev_container() {
         -t "${DEV_IMAGE}"
 }
 
-main "$@"
\ No newline at end of file
+main "$@"
```

Comment:
Even though it is trivial, can we remove this change to not affect the file history?

Reply:
done